### DISCUSSION TITLE
DNS-based Authentication of Named Entities - RFC 6698

### DISCUSSION THREAD

**Discussion by DtaLeech** [nginx-proxy/nginx-proxy#2426] [Ideas]
Hello folks,
I wanted to ask if you have thought about supporting DNS-based Authentication of Named Entities for TLS?

More and more people are using DNSSEC and with the TLSA-Record we could have an alternative to letsencrypt. Here we could get some inspiration: https://github.com/buffrr/letsdane

I know it would take some work to implement a DNS update service.
But its inspiration is already in the world: https://github.com/dschaper/docker-letsencrypt-nginx-proxy-companion

Cheers

**Comment by buchdag** [comment]
Hi. I'm not familiar with RFC6698 and I'm having a hard time understanding the feature being requested here.
